portfolio/src/main/webapp/script.js

```diff
@@ -83,7 +83,7 @@ function addRandomFact(){
 }
 
 function getMessages(){
-    fetch('/data').then(response => response.json()).then((comment) => {
+    fetch('/data').then(response => response.json()).then((comments) => {
```

Comment:
indentation looks off here. The continuation from the promise should be +2 indented from the `fetch`

Reply:
Fixed